spharpy/samplings/samplings.py

```diff
@@ -1,30 +1,48 @@
 """
 Collection of sampling schemes for the sphere
 """
-import urllib3
+import os
+from urllib3.exceptions import InsecureRequestWarning
 import numpy as np
-from spharpy.samplings.coordinates import Coordinates, SamplingSphere
+from spharpy.samplings.coordinates import SamplingSphere
+from pyfar import Coordinates
 import spharpy
+import warnings
+import scipy.io as sio
+import requests
+from multiprocessing.pool import ThreadPool
 
-from ._eqsp import point_set as eq_point_set
+from ._eqsp import point_set, lebedev_sphere
```

Comment:
point_set is rather arbitrary, I'd prefer changing that back to eq_point_set

Reply:
all other things are obsolute, becasue we will kick out the spharpy Coordinates anyway, right?